# Get a servo encoder filter's current duty. 
int32 servo_id
---
int32 count
